libraries/typescript/index.js

```diff
@@ -0,0 +1,2 @@
+module.exports = require(process.env.TYPESCRIPT_PATH ||
+	__dirname + "/../shipped-typescript");
```

Comment:
This is where the magic happens: use the environment variable or fall back the shipped typescript version (current behaviour)